### PR TITLE
feat: add InitPriceFeedIndex method IDL

### PR DESCRIPTION
**Pull request by ali-bahjati**
This PR adds the idl for [this](https://github.com/pyth-network/pyth-client/pull/416) change in the oracle program.

### CODE REVIEW

File: src/anchor/idl.json [hunk 1]
```diff
@@ -640,6 +640,42 @@
...
+          "isSigner": true
+        },
+        {
+          "name": "priceAccount",
+          "isMut": true,
+          "isSigner": false
+        },
+        {
+          "name": "permissionsAccount",
+          "isMut": true,
```
Comment (Riateche) [inline src/anchor/idl.json:660]: We also need to change `permissionsAccount` to be writable in the declaration of `addPrice`.

File: src/anchor/idl.json [hunk 2]
```diff
@@ -640,6 +640,42 @@
...
+                "value": "permissions"
+              }
+            ]
+          }
+        },
+        {
+          "name": "systemProgram",
+          "isMut": false,
+          "isSigner": false
+        }
```
Comment (Riateche) [inline src/anchor/idl.json:676]: It doesn't need the system program anymore. Looks like I forgot to update the comment in one of the places.

**Review by Riateche — CHANGES_REQUESTED** [review]

File: src/anchor/idl.json (hunk 1 above)
Comment (ali-bahjati) [reply src/anchor/idl.json:660]: Can you also update the docs for those methods in your PR? it seems they are deprecated.

**Review by Riateche — APPROVED** [review]

File: src/anchor/idl.json (hunk 1 above)
Comment (Riateche) [reply src/anchor/idl.json:660]: Updated.

**Pull request merged**